docs/faq.md

```diff
@@ -59,3 +59,20 @@ Each queue enforces a limit on the percentage of resources allocated to a user a
 ### Q: How to configure virtual cluster capacity? 
 
 A: Please refer [configure virtual cluster capacity](../pai-management/doc/how-to-write-pai-configuration.md#configure_vc_capacity)
+
+### Q: How to use private docker registry job image when submitting an OpenPAI job? 
+
+A: Please refer [job_tutorial.md](./job_tutorial.md) to config the auth file at job submit json file:
```

Comment:
please give a concrete example of a json file containing the "authFile" field.

Reply:
added